src/components/modules/nutrition/add-food/food-list.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+    <v-list two-line>
+        <template v-for="(food, index) in foods">
+            <food-list-item :key="index" :food="food">
+            </food-list-item>
+        </template>
+    </v-list>
+</template>
+
+<script lang="ts">
+import { Component, Vue, Prop, Watch } from 'vue-property-decorator';
+
+import FoodListItem from './food-list-item.vue';
+
+import { FoodListItem as FoodListItemModel } from './food-list-item.model';
+
+@Component({
+    components: {
+        'food-list-item': FoodListItem
```

Comment:
Why component's name in kebab case? In other file I've seen camel case.

Reply:
I think there is everywhere kebab case, but I will check

Reply:
Done